hack/update-checksum-windows.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+YAMLS_DIR="${WORK_DIR}"/../build/yamls
+MANIFESTS=$(ls $YAMLS_DIR/antrea-windows*.yml)
+WINDOWS_DIR="${YAMLS_DIR}"/windows
+
+checksum_windows_config=$(cat ${WINDOWS_DIR}/base/conf/* | sha256sum | cut -d " " -f 1)
+
+checksum_default=$(cat ${WINDOWS_DIR}/default/conf/* | sha256sum | cut -d " " -f 1)
+
+checksum_containerd=$(cat ${WINDOWS_DIR}/containerd/conf/* | sha256sum | cut -d " " -f 1)
```

Comment:
this is a bit unpractical IMO as it will generate the wrong checksum if there are some extra files under the `conf/` directories. For example, my text editor sometimes creates a temporary back up file in the same directory when I edit a file:
```
$ ls build/yamls/windows/base/conf/antrea-
antrea-agent.conf     antrea-agent.conf~    antrea-cni.conflist   antrea-cni.conflist~
```

Why did we change from the original version which explicitly listed the files to include in the checksum computation?

If the goal is to ensure that we don't need to update the list when adding new files, we could use a command like this one:
```
checksum_windows_config=$(cd build/yamls/windows/base/conf/ && cat $(git ls-files) | sha256sum | cut -d " " -f 1)
```

Or anything else that only includes checked-in files?

Reply:
changed to explicitly list the files and added set line for error checking if any file is renamed or missing. If a new file is added it needs to be added in the list. 
I didn't prefer adding all checked-in files inside a dir because it may happen that a new file is added and  but not staged when calculating checksum. This will result in wrong checksum and will not throw any error as well, so it can go unnoticed.